packages/office-ui-fabric-react/src/components/Announced/examples/Announced.BulkOperations.Example.tsx

```diff
@@ -38,8 +38,8 @@ const _names: string[] = [
   'Makenzie Sharett'
 ];
 
-function generateRandomDate(): string {
-  return new Date(new Date(2010, 0, 1).getTime() + Math.random() * (new Date().getTime() - new Date(2010, 0, 1).getTime())).toDateString();
+function getMockDateString(): string {
+  return 'Thu Jan 05 2017‌';
```

Comment:
What is on this date? 🤔

Reply:
🤷‍♂ I pulled it from the passing CI snapshot tests for the modified `Announced` examples.

I made this static to mitigate because different dates are reported due to use of `Math.random` on CI machines vs. local. I felt this change acceptable for the time-being. Otherwise anyone who modifies these specific snapshots will face the same friction.

cc: @natalieethell